src/main/java/ch/epfl/biop/wrappers/cellpose/ij2commands/Cellpose_SegmentImgPlusOwnModelAdvanced.java

```diff
@@ -19,75 +19,89 @@
+    double flow_threshold = 0.4;
+
+    @Parameter ( label = "Anisotropy between xy and z (1 means none)" )
+    double anisotropy = 1.0;
+
+    @Parameter ( label = "Diameter threshold (default 12)" )
+    double diam_threshold = 12.0;
 
-    @Parameter (required = false)
-    File model_path= new File ("cellpose");
+    @Parameter(required = false , label = "model_path to your owm model (default cellpose for pretrained model) ")
+    File model_path = new File("cellpose");
 
-    @Parameter(choices = {  "nuclei",
+    @Parameter(choices = {"nuclei",
                             "cyto",
                             "cyto2",
-                            "cyto (no nuclei)",
+                            "cyto2_omni",
+                            "bact_omni",
                             "own model nuclei",
                             "own model cyto",
                             "own model cyto2",
-                            "own model cyto (no nuclei)",
-                            "own model cyto2 (no nuclei)"} , callback = "modelchanged")
-    String model ;
+                            "own model cyto2_ommi",
+                            "own model bact_omni"}, callback = "modelchanged")
+    String model;
 
-    /*
-    @Parameter
-    double anisotropy;
-    */
+    @Parameter (label = "nuclei_channel (set to 0 if not necessary)")
+    int nuclei_channel;
+
+    @Parameter (label = "cyto_channel (set to 0 if not necessary)")
+    int cyto_channel;
+
+    @Parameter(choices = {"2D", "3D"})
+    String dimensionMode;
+
+    @Parameter (label = "stitch_threshold (between 0 and 1, default -1)")
+    double stitch_threshold = -1;
 
     @Parameter
-    int nuclei_channel ;
+    boolean omni;
 
     @Parameter
-    int cyto_channel;
+    boolean cluster;
 
-    @Parameter(choices = {"2D","3D"})
-    String dimensionMode ;
+    @Parameter(required = false , label="add more flags")
+    String additionnal_flags;
 
-    @Parameter (type= ItemIO.OUTPUT)
-    ImagePlus cellpose_imp ;
+    @Parameter(type = ItemIO.OUTPUT)
+    ImagePlus cellpose_imp;
 
-    Boolean verbose=true ;
+    Boolean verbose = true;
 
 
     // propose some default value when a model is selected
-    public void modelchanged(){
-        if (model.equals("nuclei")){
+    public void modelchanged() {
+        if (model.equals("nuclei")) {
             nuclei_channel = 1;
-            cyto_channel = -1 ;
-        } else if ( (model.equals("cyto"))||(model.equals("cyto2")) ){
+            cyto_channel = -1;
+        } else if ((model.equals("bact_omni"))) {
             cyto_channel = 1;
-            nuclei_channel = 2 ;
-        } else if (model.equals("cyto (no nuclei)")){
+            nuclei_channel = -1;
+        } else if ((model.equals("cyto")) || (model.equals("cyto2")) || (model.equals("cyto2_omni"))) {
             cyto_channel = 1;
-            nuclei_channel = -1 ;
-        } else if (model.equals("own model nuclei")){
```

Comment:
You can put public static final String fields to avoid duplicating everywhere "cyto2", "cyto2_omni" etc

Reply:
done in https://github.com/BIOP/ijl-utilities-wrappers/pull/10/commits/93126d6876208f560ab62dd5e510e50adaacfc3c